From 7c41be5d7a580fde67b9d05700c72eb10403a95f Mon Sep 17 00:00:00 2001
From: Nam Nguyen <nam.nguyen.yh@renesas.com>
Date: Wed, 26 Oct 2022 04:12:19 +0700
Subject: [PATCH 1/3] samples: renesas_taurus_client: Adding License for
 Renesas Taurus client sample driver

Some file source of Renesas Taurus client sample driver were missing license,
this patch is for adding them.

Signed-off-by: Nam Nguyen <nam.nguyen.yh@renesas.com>
---
 samples/renesas_taurus_client/r_taurus_bridge.h        | 6 ++++++
 samples/renesas_taurus_client/r_virtdev_drv_api_priv.h | 6 ++++++
 samples/renesas_taurus_client/taurus_client_sample.c   | 7 +++++++
 3 files changed, 19 insertions(+)

diff --git a/samples/renesas_taurus_client/r_taurus_bridge.h b/samples/renesas_taurus_client/r_taurus_bridge.h
index 6d3db7df4517..e030d9e005e2 100644
--- a/samples/renesas_taurus_client/r_taurus_bridge.h
+++ b/samples/renesas_taurus_client/r_taurus_bridge.h
@@ -1,3 +1,9 @@
+/* SPDX-License-Identifier: GPL-2.0-only OR MIT */
+/*
+ * r_taurus_bridge.h  --  Renesas Taurus client sample driver
+ *
+ * Copyright (C) 2022 Renesas Electronics Corporation
+ */
 
 #ifndef R_TAURUS_BRIDGE_H
 #define R_TAURUS_BRIDGE_H
diff --git a/samples/renesas_taurus_client/r_virtdev_drv_api_priv.h b/samples/renesas_taurus_client/r_virtdev_drv_api_priv.h
index a5d9eb908eb2..70e89b412585 100644
--- a/samples/renesas_taurus_client/r_virtdev_drv_api_priv.h
+++ b/samples/renesas_taurus_client/r_virtdev_drv_api_priv.h
@@ -1,3 +1,9 @@
+/* SPDX-License-Identifier: GPL-2.0-only OR MIT */
+/*
+ * r_virtdev_drv_api_priv.h  --  Renesas Taurus client sample driver
+ *
+ * Copyright (C) 2022 Renesas Electronics Corporation
+ */
 
 #ifndef R_VIRTDEV_DRV_API_PRIV_H
 #define R_VIRTDEV_DRV_API_PRIV_H
diff --git a/samples/renesas_taurus_client/taurus_client_sample.c b/samples/renesas_taurus_client/taurus_client_sample.c
index a0f89650a8a2..f7e4ed3a033e 100644
--- a/samples/renesas_taurus_client/taurus_client_sample.c
+++ b/samples/renesas_taurus_client/taurus_client_sample.c
@@ -1,3 +1,10 @@
+// SPDX-License-Identifier: GPL-2.0-only OR MIT
+/*
+ * taurus_client_sample.c  --  Renesas Taurus client sample driver
+ *
+ * Copyright (C) 2022 Renesas Electronics Corporation
+ */
+
 #define DEBUG
 
 #include <linux/kernel.h>
-- 
2.17.1

